src/main/java/org/janelia/saalfeldlab/paintera/PainteraDefaultHandlers.java

```diff
@@ -441,11 +447,32 @@ public static void grabFocusOnMouseOver(final Node node)
 
 	public void toggleInterpolation()
 	{
-		final Source<?> source = sourceInfo.currentSourceProperty().get();
-		if (source == null) { return; }
-		final ObjectProperty<Interpolation> ip = sourceInfo.getState(source).interpolationProperty();
-		ip.set(ip.get().equals(Interpolation.NLINEAR) ? Interpolation.NEARESTNEIGHBOR : Interpolation.NLINEAR);
-		baseView.orthogonalViews().requestRepaint();
+		if (globalInterpolationProperty.get() != null)
+		{
+			globalInterpolationProperty.set(globalInterpolationProperty.get().equals(Interpolation.NLINEAR) ? Interpolation.NEARESTNEIGHBOR : Interpolation.NLINEAR);
```

Comment:
At some point we should use [`Interpolation.next`](https://github.com/bigdataviewer/bigdataviewer-core/commit/c9e036b3441047653693ddb176124ad0e75b3cf6) instead of cycling through interpolations manually. That requires update of BDV dependency version, which might break some code, so this should happen separately.